core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -656,6 +656,8 @@
       "1.4.0"),
   TSERV_BULK_TIMEOUT("tserver.bulk.timeout", "5m", PropertyType.TIMEDURATION,
       "The time to wait for a tablet server to process a bulk import request.", "1.4.3"),
+  TSERV_HEALTH_CHECK_FREQ("tserver.health.check.interval", "30m", PropertyType.TIMEDURATION,
```

Comment:
As follow on work may be good to investigate if anything else should use this property.  There is a check that perodically looks for stuck compactions in the tablet server, maybe it could use this property too.

Reply:
Any idea where this other check is located?

Reply:
Is it `CompactionWatcher`? I noticed this method never gets called: https://github.com/apache/accumulo/blob/7fb1b5df9e3abe18304039fb3e781b2617eec4b5/server/base/src/main/java/org/apache/accumulo/server/compaction/CompactionWatcher.java#L124

Reply:
The `run()` method does get called though... I am not sure if `startWatching()` should be called instead though.

Reply:
The `startWatching()` method does get called, once in the constructor of `MajorCompactor`. And it looks like the other times that the `run()` gets called are just for single, one time checks.
https://github.com/apache/accumulo/blob/dca3ae8d056a24510aa9d86347d9d6338613c4b3/server/compactor/src/main/java/org/apache/accumulo/compactor/Compactor.java#L757-L759